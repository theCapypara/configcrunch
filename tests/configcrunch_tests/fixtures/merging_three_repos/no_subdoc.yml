base:
  $ref: /base_ref
  more:
    base: base
    three_base: base