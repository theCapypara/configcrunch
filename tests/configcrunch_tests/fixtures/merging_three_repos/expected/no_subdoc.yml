base:
  more:
    base: base
    one: one
    two: two
    three: three
    one_two: two
    two_three: three
    three_base: base